src/briefcase/platforms/linux/system.py

```diff
@@ -830,26 +847,25 @@ class LinuxSystemPackageCommand(LinuxSystemMixin, PackageCommand):
     def packaging_formats(self):
         return ["deb", "rpm", "pkg", "system"]
 
-    def _verify_deb_tools(self):
-        """Verify that the local environment contains the debian packaging tools."""
-        if not Path("/usr/bin/dpkg-deb").exists():
+    def _verify_packaging_tools(self, app: AppConfig):
+        """Verify that the local environment contains the packaging tools."""
+        if install_cmd := self._system_requirement_tools(app)[2]:
+            install_cmd = " ".join(install_cmd)
+
+        if app.packaging_format == "deb" and not Path("/usr/bin/dpkg-deb").exists():
             raise BriefcaseCommandError(
-                "Can't find the dpkg tools. Try running `sudo apt install dpkg-dev`."
+                f"Can't find the dpkg tools. Try running `sudo {install_cmd} dpkg-dev`."
             )
 
-    def _verify_rpm_tools(self):
-        """Verify that the local environment contains the redhat packaging tools."""
-        if not Path("/usr/bin/rpmbuild").exists():
+        if app.packaging_format == "rpm" and not Path("/usr/bin/rpmbuild").exists():
             raise BriefcaseCommandError(
-                "Can't find the rpm-build tools. Try running `sudo dnf install rpm-build`."
+                f"Can't find the rpm-build tools. Try running `sudo {install_cmd} rpm-build`."
             )
 
-    def _verify_pkg_tools(self):
-        """Verify that the local environment contains the arch packaging tools(ABS)."""
-        if not Path("/usr/bin/makepkg").exists():
+        if app.packaging_format == "pkg" and not Path("/usr/bin/makepkg").exists():
             raise BriefcaseCommandError(
-                "Can't find the `makepkg` tool. Try running `sudo pacman -Syu pacman`."
```

Comment:
bit of chicken and egg problem here....gotta use `pacman` to install `pacman`.... 🤔